packages/authentication/src/__tests__/unit/providers/authentication.provider.unit.ts

```diff
@@ -110,7 +110,10 @@ describe('AuthenticateActionProvider', () => {
       strategy.setMockUser(mockUser);
       provider = new AuthenticateActionProvider(
         () => Promise.resolve(strategy),
-        u => (currentUser = u),
+        (...u) => {
+          currentUser = u[0] as UserProfile;
```

Comment:
Is it safe to assume `u` has at least one element? I think `u` could be empty.

Reply:
This is just a mock-up for the test. Even when `u` is an empty array, `u[0]` returns `undefined`. The setter should be always injected for regular usage.

Reply:
Oh I see it's in a test file.